app/src/main/java/com/paypal/psix/fragments/EventStatusFragment.java

```diff
@@ -40,6 +42,9 @@ public View onCreateView(final LayoutInflater inflater, ViewGroup container,
         eventTitle.setText(event.name);
         eventDateHeader.setText(event.getFormattedDate());
         Picasso.with(getActivity()).load(event.imageURL).into(eventImageHeader);
+        rsvpsAdapter = new RsvpsAdapter(getActivity(), new ArrayList<>(event.rsvps()));
```

Comment:
Since java 7 you have the "diamond operator" <> which determines the dynamic generic type from the static generic type.